packages/smart-contracts/hardhat.config.ts

```diff
@@ -100,6 +105,10 @@ const setExplorerApiKey = (hre: HardhatRuntimeEnvironment) => {
       hre.config.etherscan.apiKey = process.env.FTMSCAN_API_KEY;
       return;
     }
+    case 'velas': {
+      hre.config.etherscan.apiKey = process.env.VELAS_EVM_EXPLORER_API_KEY;
```

Comment:
Velas doesn't use EtherScan based explorer but Blockscout so this added case is not needed.